### PR TITLE
Fix baiyesian AB test failing with NA values

### PR DESCRIPTION
**Pull request by shun2wang**
Fix https://github.com/jasp-stats/jasp-issues/issues/2395

### CODE REVIEW

**Review by boutinb — APPROVED** [review]

**Comment by boutinb** [comment]
It works for me. @vandenman ?

**Comment by shun2wang** [comment]
Well this fix not safety for some conditions.

File: R/abtestbayesian.R [hunk 1]
```diff
@@ -68,15 +68,15 @@ ABTestBayesianInternal <- function(jaspResults, dataset = NULL, options) {
   #   The (numeric) columns given as dependent/covariates/wlsWeights
 
   if (!is.null(dataset))
-    return(dataset)
+    return (na.omit(dataset))
 
   asNum             <- c(options$n1, options$y1, options$n2, options$y2)
   dataset           <- .readDataSetToEnd(columns.as.numeric = asNum, excluse.na.listwise = asNum)
```
Comment (vandenman) [inline R/abtestbayesian.R:74]: shouldn't `excluse.na.listwise` be `exclude.na.listwise`? That automatically remove the NAs.

Comment (shun2wang) [reply R/abtestbayesian.R:74]: ah, never mind. I missed it.

**Review by vandenman — APPROVED** [review]

**Comment by vandenman** [comment]
I retriggered the macOs test, feel free to merge when it passes.

**Comment by shun2wang** [comment]
Thanks! I have no permission to merge:-)

**Pull request merged**